Std/Logic.lean

```diff
@@ -649,6 +651,8 @@ theorem Decidable.or_congr_left' [Decidable c] (h : ¬c → (a ↔ b)) : a ∨ c
 theorem Decidable.or_congr_right' [Decidable a] (h : ¬a → (b ↔ c)) : a ∨ b ↔ a ∨ c := by
   rw [or_iff_not_imp_left, or_iff_not_imp_left]; exact imp_congr_right h
 
+alias ⟨Decidable.or_not_not_of_not_and, _⟩ := Decidable.not_and_iff_or_not
+
```

Comment:
This seems like a niche use.  I'm not in favor putting these definitions here.

One side note, I don't see a reason to prefer `Decidable.not_and_iff_or_not` over `not_and`.  Am I missing something?

Reply:
Are you asking about the `iff_or_not` suffix (which I agree could be dropped), or the `Decidable` namespace (which I think is important to distinguish that it is not a classical theorem and takes hidden decidability arguments).